### PR TITLE
Fix x64/Debug build on MSVS

### PR DESCRIPTION
**Pull request by tbennun**
This fixes building glog on Windows using the Debug configuration on the x64 platform.

### CODE REVIEW

**Comment by pphaneuf** [comment]
I cannot find a CLA (contribution license agreement) on file for you? We cannot accept your contribution without one, unfortunately...

Could you make sure you have one by going here?

https://cla.developers.google.com/

Thanks!

**Comment by pphaneuf** [comment]
Dummy comment to trigger the CLA check...

**Comment by tbennun** [comment]
@pphaneuf I submitted it just now.

**Comment by pphaneuf** [comment]
And the newfangled CLA robot saw it. Thanks!

I'm not reviewing the code change, I have no idea about this Windows stuff, just dropping by for the CLA stuff, leaving it to the other maintainers to merge this in.

File: src/logging.cc [hunk 1]
```diff
@@ -1459,7 +1459,7 @@ static void logging_fail() {
 #if defined(_DEBUG) && defined(_MSC_VER)
   // When debugging on windows, avoid the obnoxious dialog and make
   // it possible to continue past a LOG(FATAL) in the debugger
-  _asm int 3
+  __debugbreak();
```
Comment (ukai) [inline src/logging.cc]: no need to include <intrin.h> ?
https://msdn.microsoft.com/en-us/library/f408b4et.aspx

Comment (tbennun) [inline src/logging.cc]: It compiles without it. 
As far as I know, <windows.h> includes it, so maybe windows/port.h is included somehow.

To be sure, the function "DebugBreak()" from windows.h can be used:
https://msdn.microsoft.com/library/windows/desktop/ms679297.aspx

However, the official replacement (according to https://msdn.microsoft.com/en-us/library/45yd4tzz.aspx) for _asm int 3 is __debugbreak().

Comment (ukai) [inline src/logging.cc]: ok.
lgtm

**Pull request merged**